openai/api_resources/abstract/api_resource.py

```diff
@@ -1,11 +1,15 @@
 from urllib.parse import quote_plus
 
 from openai import api_requestor, error, util
+import openai
 from openai.openai_object import OpenAIObject
+from openai.util import ApiType
 
 
 class APIResource(OpenAIObject):
     api_prefix = ""
+    azure_api_prefix = 'openai/deployments'
+    azure_api_version = '?api-version=2021-11-01-preview'
```

Comment:
I think we should be able to just always pass in an api_version to the query param and have it be ignored on openai side for now?

My understanding is longer term we probably want to just use that anyway to avoid/communicate breaking changes.

Reply:
Ok, sure